ops/helmfiles/config/testnet.yaml

```diff
@@ -35,12 +35,33 @@ collator:
       operator: Equal
       value: "true"
       effect: NoSchedule
-  firstCollatorNodeSelector:
-    production: "true"
-    topology.kubernetes.io/zone: europe-west1-b
-  secondCollatorNodeSelector:
-    production: "true"
-    topology.kubernetes.io/zone: europe-west1-c
+  # Node specific collator configs
+  collator01:
+    # Identity ID: 12D3KooWAfm5cK7DuUa2zh9SrSoHr2fsXXYbLALWXs5ob9mTDKHJ
+    customNodeKey: '5ad94653edf592f59d1b54960cda1334596cc3dcab8dd96adb334d3e22e912ef'
```

Comment:
This node key seems to be repeated...
Please do not repeat any node key, please make sure all node keys are unique